src/components/EntityInfo.vue

```diff
@@ -1,7 +1,6 @@
 <template>
   <div
     id="entity-info"
-    :class=" { 'staff': isRoleStaff, 'hover': showHoverStyle }"
```

Comment:
Since you're getting rid of the hover class, you can delete the property on lines 64-65 and the style on lines 124-126.